.storybook/recipes/GlobalHeader/GlobalHeader.module.css

```diff
@@ -42,27 +42,41 @@
  *    on small screens, but is hidden when the nav
  *    can be exposed
  */
-.global-header__menu-button {
-  @media all and (min-width: $eds-bp-lg) {
+.global-header__menu-button.global-header__menu-button {
+  transition: background var(--eds-anim-fade-quick) var(--eds-anim-ease);
+
+  &:hover,
+  &:focus {
+    background: var(--eds-color-brand-grape-800);
```

Comment:
```suggestion
    background: var(--eds-theme-color-text-link-strong-hover);
```

Reply:
Courtney and I were talking about how to handle these. `text-link` refers to actual text. Because this is technically a background, we may want to use the `button-icon` token for this. One of those weird background hover examples.

Reply:
Added a TODO for these until we come to a clear decision with design around this